exercise/appeaes_more_than_once.py

```diff
@@ -0,0 +1,24 @@
+"""
+This module contains a code for exercises 10-7 related to:
+Think Python, 2nd Edition
+Chapter 10: Lists
+
+Takes a list and returns True if there is
+any element that appears more than once.
+"""
+
+def has_duplicates():
+    t = list(input('Enter list:'))
+    set_t = set(t)
+    """
+    Input a list 
+
+    Return True if list has any element appears more than once.
+    """
```

Comment:
With the function docstring, always place it at the beginning of the function body.
(It should be below line number 10)